modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -44,11 +44,20 @@ class IgniteAwareService(BackgroundThreadService):
     def __init__(self, context, num_nodes, version, properties):
         super(IgniteAwareService, self).__init__(context, num_nodes)
 
+        if 'project' in context.globals:
+            self.path = IgnitePath(context.globals['project'])
+        else:
+            self.path = IgnitePath()
+
+        if 'jvm_opts' in context.globals:
+            self.jvm_options = context.globals['jvm_opts']
+        else:
+            self.jvm_options = ""
```

Comment:
Seems, you may just always read from context.globals['jvm_opts']?

Reply:
It will raise KeyError in case that jvm_options aren't specified.

Reply:
Python dict has method get(key, default)

Reply:
fixed